### DISCUSSION TITLE
Marking queued commands(messages)

### DISCUSSION THREAD

**Discussion by Wolverator** [Rapptz/discord.py#9550] [Q&A] [answered]
Hello!
I have a bot that has commands A, B and C marked as `commands.group` and marked with `commands.max_concurrency(number=1, per=commands.BucketType.default, wait=True)`, so bot is able to process only one such command at a time, wherever command is asked from. And i'm very grateful that this feature has `wait` option so users' commands will be executed, but in order (queue).

But my question is: can i make my bot react to messages that are placed in this queue with time emoji like `🕑`, to show user to wait until this command will get it's turn to be done?

I tried to do basic way ( wrote `await ctx.message.add_reaction('🕑')` as first line of code inside command itself) but i've found that even this line will not be executed while command is waiting in queue. 😅 
Thus, i'm here and asking this question =)

**Comment by Rapptz** [comment, marked as answer]
Unfortunately this isn't really possible. `max_concurrency` is basically the first thing called, the only thing that comes before are checks but adding a reaction there is bad form and wouldn't do what you want anyway.

**Reply by Wolverator** [reply]
So if i want adding reactions to queued commands, i have to implement my own internal queue system that will allow me that.
Got it! Thanks for fast reply ^_^
